packages/flame_jenny/example/lib/main.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:dashbook/dashbook.dart';
+import 'package:flutter/widgets.dart';
+import 'package:jenny_example/stories/advanced_jenny.dart';
+import 'package:jenny_example/stories/simple_jenny.dart';
+
+void main() {
+  runAsDashbook();
+}
+
+void runAsDashbook() {
+  final dashbook = Dashbook(
+    title: 'Jenny Examples',
+  );
+
+  addJennySimpleExample(dashbook);
+  addJennyAdvancedExample(dashbook);
+
+  runApp(dashbook);
+}
```

Comment:
Just keep the simple example in here and no dashbook, and have both examples inside of the root examples directory instead, create a directory called `flame_jenny` here and follow the same structure as the other examples:
https://github.com/flame-engine/flame/tree/main/examples/lib/stories/bridge_libraries

Reply:
Moved in [721959c](https://github.com/flame-engine/flame/pull/3086/commits/721959c28e1dc4b92c5ecf5d4dfde0556e672d05)